server_extensions_api/src/main/java/com/generalbytes/batm/server/extensions/ITransactionListener.java

```diff
@@ -166,7 +166,7 @@ default boolean isTransactionApproved(IDepositRequest depositRequest) {
      * @param request The transaction request initialized by server
      * @return {@link ITransactionRequest} that may contain modified transaction request.
      */
-    default IDepositRequest overrideTransactionRequest(IDepositRequest request) {
+    default IDepositRequest overrideDepositRequest(IDepositRequest request) {
```

Comment:
For consistency, I would name the parameter as `depositRequest`. Or just `request` in the `isDepositApproved` method.